storage/mysql/tree_storage.go

```diff
@@ -39,9 +39,9 @@ const (
 		 VALUES(?,?,?,?,?,?)`
 
 	selectSubtreeSQL = `
- SELECT x.SubtreeId, x.MaxRevision, Subtree.Nodes
+ SELECT x.TreeId, x.SubtreeId, x.MaxRevision, Subtree.Nodes
```

Comment:
Do you need to select `x.TreeId` or is just including it in the join sufficient?

Reply:
I don't think I can reference it in the join if it's not part of the X result set. If you feel like experimenting then feel free to have a go at it.

Reply:
I've just tested it and you indeed don't need `x.TreeId` here. You only need it in the inner SELECT and on the join.

Reply:
Done.